src/style.css

```
@@ -0,0 +1,148 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+
+  --moyo-color: #00acdc;
+  --reset: 0;
+  --width-content: 166px;
+  --text-color-1: #060b35;
+  --text-color-2: #616070;
+  --white-color: #fff;
+}
+
+.main {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
```

Add box-sizing: border-box, so sizes includes borders too
```suggestion
  width: 200px;
```